webui/src/pages/setup/userConfiguration.tsx

```diff
@@ -60,20 +55,13 @@ export const UserConfiguration: FC<UserConfigurationProps> = ({
                             <Form.Control type="text" value={adminUser}  onChange={handleAdminUserChange} placeholder="Admin Username" autoFocus/>
                         </Form.Group>
 
-                            <Card.Text>
-                            Please provide your email address, and indicate what optional communications you&apos;d like to receive:
-                            </Card.Text>
                             <Form.Group controlId="user-email" className="mt-4">
                                 <Form.Label>Email <span className="required-field-label">*</span></Form.Label>
                                 <Form.Control type="email" placeholder="name@company.com" value={userEmail}  onChange={handleEmailChange} />
                             </Form.Group>
 
-                            <Form.Group controlId="updates-check" className="mt-4">
```

Comment:
Please change from 
`Please provide your email address, and indicate what optional communications you&apos;d like to receive:`
to 
`...`

Reply:
Done